api/http/handler/webhooks/handler.go

```diff
@@ -0,0 +1,37 @@
+package webhooks
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	portainer "github.com/portainer/portainer"
+	httperror "github.com/portainer/portainer/http/error"
+	"github.com/portainer/portainer/http/security"
+)
+
+// Handler is the HTTP handler used to handle webhook operations.
+type Handler struct {
+	*mux.Router
+	WebhookService  portainer.WebhookService
+	EndpointService portainer.EndpointService
+	requestBouncer  *security.RequestBouncer
+}
+
+// NewHandler creates a handler to manage settings operations.
+func NewHandler(bouncer *security.RequestBouncer) *Handler {
+	h := &Handler{
+		Router:         mux.NewRouter(),
+		requestBouncer: bouncer,
+	}
+	h.Handle("/webhooks",
+		bouncer.RestrictedAccess(httperror.LoggerHandler(h.webhookCreate))).Methods(http.MethodPost)
+	h.Handle("/webhook/{id}",
```

Comment:
I believe that we don't need the `webhookInspect` operation at the moment, I'm not sure where it would be required in the front-end.